code/modules/admin/verbs/adminhelp.dm

```diff
@@ -11,6 +11,12 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 	if(prefs.muted & MUTE_ADMINHELP)
 		to_chat(src, "<font color='red'>Error: Admin-PM: You cannot send adminhelps (Muted).</font>")
 		return
+	//
+	if(isnewplayer(src))

```

Comment:
Would src be a client? Not 100% sure if this works

Reply:
Yep src would be the client the proc is called for, so the person adminhelping